Watchman.DomainModel/Mute/Queries/GetMuteEventsFromBaseQuery.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Watchman.Cqrs;
+
+namespace Watchman.DomainModel.Mute.Queries
+{
+    public class GetMuteEventsFromBaseQuery : IQuery<GetMuteEventsFromBaseQueryResult>
```

Comment:
```suggestion
    public class GetMuteEventsQuery : IQuery<GetMuteEventsQueryResult>
```
domenie nie mówisz jak dokładnie coś ma być zrobione
tylko mówisz "czego potrzebujesz" a ona już ci to załatwia

warstwy aplikacji traktuj jak używanie zewnętrznych api
nie interesuje cie czy ktoś trzyma coś w bazie danych czy pliku tekstowym
interesuje cie tylko to, czy dostaniesz te dane których potrzebujesz